src/pages/host/HostCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { ApolloError } from "apollo-client";
+import { Skeleton } from "antd";
+import { ErrorWrapper } from "components/ErrorWrapper";
+import Card from "@leafygreen-ui/card";
+import styled from "@emotion/styled/macro";
+import { css } from "@emotion/core";
+
+interface Props {
+  error: ApolloError;
+  loading?: boolean;
+  metaData: boolean;
+}
+interface StylingProps {
+  metaData: boolean;
+}
+
+export const HostCard: React.FC<Props> = ({
+  children,
+  error,
+  loading,
+  metaData,
+}) => (
+  <SiderCard metaData={metaData}>
+    {loading && !error && (
+      <Skeleton active title={false} paragraph={{ rows: 4 }} />
+    )}
+    {error && !loading && (
+      <ErrorWrapper data-cy="metadata-card-error">{error.message}</ErrorWrapper>
+    )}
+    {!loading && !error && children}
+  </SiderCard>
+);
+
+export const metaDataCard = css`
+  > * {
```

Comment:
avoid this selector in css as it is not performant

Reply:
changed it to > p